app/src/main/res/xml/advanced_settings.xml

```diff
@@ -45,6 +45,12 @@
         android:title="@string/settings_category_security"
         app:iconSpaceReserved="false">
 
+        <SwitchPreference
```

Comment:
We should not forget to hide this if its not supported.

Reply:
Actually I did not find an easy way to detect if it is supported.